bootstrap_cfn/config.py

```diff
@@ -92,6 +76,10 @@ def process(self):
         template['Outputs'] = {}
         for t in output_templates:
             template['Outputs'].update(json.loads(pkgutil.get_data('bootstrap_cfn', t)))
+        if 'includes' in self.data:
+            for inc_path in self.data['includes']:
+                inc = json.load(open(inc_path))
+                utils.dict_merge(template, inc)
```

Comment:
Ugh, I don't want to think to hard about why this seems to work because thinking about recursive functions always gives me a headache! I think there might be an edge case where it doesn't work if you don't assign the return value though so I will change it.